qemu/tests/cfg/win_irq_check.cfg

```diff
@@ -0,0 +1,22 @@
+- win_irq_check: single_driver_install.with_viorng single_driver_install.with_balloon
+    type = win_irq_check
+    only Windows
+    kill_vm = yes
+    login_timeout = 360
+    msinfo_cmd = start /wait msinfo32 /report c:\msinfo.txt /categories +resourcesirqs
```

Comment:
/categories +resourcesirqs should not work,please have a check.

Reply:
https://support.microsoft.com/en-us/help/300887/how-to-use-system-information-msinfo32-command-line-tool-switches
/categories only work in winxp,and we do not test winxp at all,so please remove the parameters if it is not supported even though it's not harmful.

Reply:
Agree.
Thanks.
I have updated.